plugins/inputs/powerdns_recursor/README.md

```diff
@@ -16,8 +16,22 @@ the unix controlsocket.
   # socket_dir = "/var/run/"
   ## Socket permissions for the receive socket.
   # socket_mode = "0666"
+
+  ## The version of the powerdns control protocol to use. You will have to
+  ## change this based on your PowerDNS Recursor version, see below:
+  ## PowerDNS <4.5.0 => Protocol Version 1
+  ## PowerDNS 4.5.0 - 4.5.9 => Protocol Version 2
+  ## PowerDNS >=4.6.0 => Protocol Version 3
+  ## By default this is set to 1.
+  # control_protocol_version = 1
 ```
 
+### Newer PowerDNS Recursor versions
+
+The control protocol was updated in PowerDNS Recursor 4.5.0. If you're using
+version 4.5.0 and newer you need to set `new_control_protocol` to `true` in
```

Comment:
```suggestion
version 4.5.0 and newer you need to set `control_protocol_version` to `3` in
```

Reply:
@OrfeasZ I do agree with this change, better sounding config option. Also may way to mention version 2 was briefly used.

Reply:
Looks like I missed updating this part of the readme. I'll do a revision.